forem/forem
# Make home link an admin-customisable navigation link

## What type of PR is this? (check all applicable)


## Description

This changes the "Home" link that appears in the left sidebar to be a "Navigation Link", customisable/removable in the admin area.

I'd appreciate some backend eyes on this to make sure all is well with both new forems and forems that will get the new update deployed (based on the work done in https://github.com/forem/forem/pull/16112 and local testing, I believe we're all good). 

## Related Tickets & Documents

Closes https://github.com/forem/forem/issues/16076

## QA Instructions, Screenshots, Recordings

- Run the branch locally and check the "Home" link still appears in the sidebar
- Go to the navigation links admin area: `/admin/customization/navigation_links`
- Experiment with editing the Home link and check updates are reflected in the UI
- Delete the Home link and check it's now gone from the sidebar

### UI accessibility concerns?

N/A

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_
